pytest_tests/testsuites/object/test_object_api.py

```diff
@@ -337,6 +341,69 @@ def test_object_search_should_return_tombstone_items(
                     object_type == "TOMBSTONE"
                 ), f"Object wasn't deleted properly. Found object {tombstone_oid} with type {object_type}"
 
+    @allure.title("Validate objects search by common prefix")
+    def test_search_object_api_common_prefix(self, default_wallet: str, simple_object_size: int):
+        FILEPATH_ATTR_NAME = "FilePath"
+        cli = NeofsCli(self.shell, NEOFS_CLI_EXEC, WALLET_CONFIG)
+        wallet = default_wallet
+        cid = create_container(wallet, shell=self.shell, endpoint=self.cluster.default_rpc_endpoint)
```

Comment:
Wouldn't it be better to do container creation in fixture?

Reply:
Perhaps, but for some reason containers are not deleted in this test suite (maybe it consumes time and is not really needed cause doesn't affect other tests), so the fixture will be useless, just with one statement for creation and without a proper cleanup.

Reply:
> Perhaps, but for some reason containers are not deleted in this test suite (maybe it consumes time and is not really needed cause doesn't affect other tests), so the fixture will be useless, just with one statement for creation and without a proper cleanup.

Maybe we should start deleting containers in this test suite?
Not deleting containers and other test files in tests is many places in the code, it's a very bad practice, let's make the test code better.

Reply:
fixed for this test